doc/user/release-notes.md

```diff
@@ -0,0 +1,18 @@
+---
+title: "Release Notes"
+description: "What's new in this version of Materialize"
+menu: "main"
+weight: 500
+---
+
+This page details changes between versions of Materialize, including:
+
+- New features
+- Major bug fixes
+- Substantial API changes
+
+For information about available versions, see our [Versions page](../versions).
+
+## 0.1.0 &rarr; 0.1.1
+
+- [`CREATE MATERIALIZED SOURCE`](../sql/create-source) let you automatically store all of a source's data in an index. Previously, you would have to create a source, and then create a materialized view that selected all of the source's content.
```

Comment:
The description of `CREATE MATERIALIZED SOURCE` is correct, but there is much more to the change. 

It is now possible to create/show/drop indexes on sources. `SHOW FULL SOURCES` will expose information on whether the source is materialized. materialized sources can be directed queried by select statements.